sympy/functions/special/tests/test_error_functions.py

```diff
@@ -682,6 +682,10 @@ def test_ci():
     assert Ci(x).series(x, oo) == -cos(x)*(-6/x**3 + 1/x \
         + O(x**(-7), (x, oo)))/x + (24/x**4 - 2/x**2 + 1 \
         + O(x**(-7), (x, oo)))*sin(x)/x
+
+    assert Ci(x).series(x, -oo) == -cos(x)*(-6/x**3 + 1/x + O(-1/x**7, (x, -oo)))/x + \
+            (24/x**4 - 2/x**2 + I*pi + 1 + O(-1/x**7, (x, -oo)))*sin(x)/x
+
```

Comment:
Wait I don't think the `I*pi` term should be there inside this bracket `(24/x**4 - 2/x**2 + I*pi + 1 + O(-1/x**7, (x, -oo)))`
I think it should be `(24/x**4 - 2/x**2 + 1 + O(-1/x**7, (x, -oo)))*sin(x)/x + I*pi`